src/main/java/mike/Parser.java

```diff
@@ -0,0 +1,132 @@
+package mike;
+
+import mike.task.Deadline;
+import mike.task.Event;
+import mike.task.Task;
+import mike.task.ToDo;
+
+/**
+ * Parses input from the user
+ */
+public class Parser {
+
+    /**
+     * Returns a Command with the enum type by parsing the input
+     *
+     * @param input String object representing the user's input
+     * @return a Command of the enum type
+     */
+    public static Command parseCommand(String input) {
+        return Command.valueOf(input.toUpperCase());
+    }
+
+    /**
+     * Returns a Task Object by parsing the description depending on the Command
+     *
+     * @param command an object of the Command type
+     * @param input the input of the user
+     * @return the Task corresponding to the user's command
+     * @throws ParseException if parsing fails
+     */
+    public static Task parseDescription(Command command, String input) throws ParseException {
```

Comment:
I agree with @yeoutzer and like how the function names are descriptive and there are header comments to the functions which improves ease of understanding.